cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -207,7 +207,7 @@ include::../indexes/drop-a-non-existing-index.asciidoc[leveloffset=+1]
 
 === List indexes
```

Comment:
First in the `Indexes for search performance` chapter there is a menu with a link that is supposed to take you here. That link does work in 4.1 but not in 4.2-preview or in your PR. So it was already broken but I happened to see it now.

Reply:
I guess it broke when Jessica moved the section from the test to this file, will fix :)